src/main/java/mate/controller/IndexController.java

```diff
@@ -1,9 +1,37 @@
 package mate.controller;
 
+import java.io.IOException;
+import java.time.LocalDateTime;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import mate.dao.MyCoolResource;
 
+@WebServlet(urlPatterns = "/index")
 public class IndexController extends HttpServlet {
     private MyCoolResource myResource;
-    //TODO: implement
+
+    @Override
+    public void init(ServletConfig config) throws ServletException {
+        myResource = MyCoolResource.openResource();
+        super.init(config);
+    }
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String currentTime = LocalDateTime.now().toString();
+        myResource.write(currentTime);
+        req.setAttribute("date", currentTime);
```

Comment:
you set attribute but you never show it on the page. remove this or show on the page, don't write useless code